src/main/res/layout/activity_microphone_configure.xml

```diff
@@ -20,100 +20,99 @@
         app:layout_constraintVertical_chainStyle="spread"
         app:popupTheme="@style/AppTheme.PopupOverlay" />
 
-    <TextView
-        android:id="@+id/userShouldDo"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:gravity="center"
-        android:text="@string/tune_the_sound_detection"
-        android:textColor="@color/colorAccent"
-        android:textSize="20sp"
-        android:textStyle="bold"
-        app:layout_constraintBottom_toTopOf="@+id/userShouldDoHint"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/toolbar"
-        app:layout_constraintVertical_weight="1" />
+        <TextView
+            android:id="@+id/userShouldDo"
```

Comment:
Remove indentation to actually correct the formatting (all structure has to match the original document). Use the spacing tools to highlight where you’re going wrong. It’s a pain I know. Good luck.